application/modules/login/controllers/login.php

```diff
@@ -83,7 +83,7 @@ public function index()
 			}
 			else
 			{
-				$data['username'] = $this->input->post('login_username');
+				$data['username'] = htmlspecialchars($this->input->post('login_username'), ENT_QUOTES, 'UTF-8');
```

Comment:
This line bothers me, on register.php (on the register module) there's a validation function/methods to prevent the fields to be some types, instead of this I'd suggest also making the same change here, and add the validation part instead of a single `htmlspecialchars` that way we're prepared for the future in terms of validations.

Reply:
There's already a function that does this in template.php I could do:

$data['username'] = $this->template->format($this->input->post('login_username'), false, false, true, false);

Reply:
for now its fine it does the job